independent-publisher-connectors/AzureADApplications/apiProperties.json

```diff
@@ -5,9 +5,10 @@
         "type": "oauthSetting",
         "oAuthSettings": {
           "identityProvider": "aad",
-          "clientId": "[[DUMMY]]",
+          "clientId": "fbe18248-e189-46d5-a4bf-b1f1b854a48c",

```

Comment:
@paulculmsee it is not allowed to have secret look alike values. Please replace it with a dummy value.

Reply:
Apologies - should have caught that one